crates/ef-testing/tests/tests.rs

```diff
@@ -123,7 +123,7 @@ mod blockchain_tests {
     // blockchain_tests!(st_static_flag_enabled, stStaticFlagEnabled); // 13 failing tests
     // blockchain_tests!(st_system_operations_test, stSystemOperationsTest); // 28 failing tests
     // blockchain_tests!(st_time_consuming, stTimeConsuming); // 14 failing tests
-    // blockchain_tests!(st_transaction_test, stTransactionTest); // failing due to: invalid length 62, expected a (both 0x-prefixed or not) hex string or byte array containing betwee
+    blockchain_tests!(st_transaction_test, stTransactionTest);
```

Comment:
This test fails due to a panic error when loading the file.
I think it's better to leave this testsuite commented out, adding the number of the issue concerned.

Reply:
@khaeljy  should i delete this PR then? if i remove the json file in the yml file and leave the testsuite commented out, then i have nothing to pull :)

Reply:
I think you can prefix the comment with `ef-test #447 - ` to indicate that an issue is linked to this test.
_(This needs the approval of someone in the team)_

Reply:
Good idea, let's do that. thank you.